spec/features/users_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+RSpec.describe User, type: :system do
+  describe 'index page' do
+    it 'I can see the username of all other users.' do
+      visit users_path
+      expect(page).to have_content('Tom')
+    end
+
+    it 'I can see the profile picture for each user.' do
+      visit users_path
+      expect(page).to have_xpath("//img[@src='https://unsplash.com/photos/F_-0BxGuVvo']")
+    end
+
+    it 'I can see the number of posts each user has written.' do
+      visit users_path
+      expect(page).to have_content('Number of posts: 0')
+    end
+
+    it 'When I click on a user Lilly, I am redirected to that user show.' do
+      visit users_path
+      click_link 'Lilly'
+      expect(page).to have_current_path(user_path(7))
+    end
```

Comment:
- [ ] There are a lot of failed tests because you suppose to create a user here to make these tests work, Please fix that.
These tests may work with you because you already created data in your database.
[Hint] You can create data and run it before each test by adding the following before your tests:
```ruby
before :each do
   User.create(name: "Tom", photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: "Teacher from Mexico")
 end
```